index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
+    <link href="https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
+    integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <meta charset="utf-8">
+    <title>Backgrounds and Gradients</title>
+  </head>
+
+  <body>
+
+    <header>
+      <!-- navegator bar here-->
+      <section>
+        <nav id="globalHeader">
+          <ul id="globalNav">
+              <li id="icon-apple">
+                <span><i class="fab fa-apple"></i></span>
+              </li><!-- mac icon-->
+              <li id="icon-store">Store</li>
+              <li id="icon-mac">Mac</li>
+              <li id="icon-iPod">iPod</li>
+              <li id="icon-iPhone">iPhone</li>
+              <li id="icon-iPad">iPad</li>
+              <li id="icon-iTunes">iTunes</li>
+              <li id="icon-support">Support</li>
+          </ul>
+            <div class="search-Nav">
+              <form class="searchForm" action="index.html" method="get">
+                <div class="searchIcon"><i class="fas fa-search"></i></div>
+                <input type="text" id="searchInput" size="6">
+              </form>
+            </div>
+        </nav>
+      </section>
+    </header>
+
+    <main>
+          <div class="background">
+        <section class="mainWrapper">
+          <h1 class="mainTitle">What will your verse be?</h1>
+          <h2 class="mainDescription row1">Everyone has something to share. See how people are using</h2>
+          <h2 class="mainDescription row2">iPad in their unique ways to add to the world’s story.</h2>
+          <div class="main link">
+          <a href="#">
+            <p id="linkText">Watch the “Your Verse” film and explore the stories <i class="far fa-play-circle"></i></p>
+          </a></div>
+        </section>
```

Comment:
Edit the section content following this guidelines 
- [ ] There should be three links
- [ ] Each link is an <a> HTML tag, with a <div> tag inside it.
- [ ] The <div> tag  inside the <a> tag should use an image as background, the text on the original page is actually an image